be/src/vec/sink/mysql_result_writer.cpp

```diff
@@ -136,7 +136,13 @@ Status VMysqlResultWriter::_add_one_column(const ColumnPtr& column_ptr,
                 if (data->is_null_at(j)) {
                     buf_ret = _buffer.push_string("NULL", strlen("NULL"));
                 } else {
-                    buf_ret = _add_one_cell(data, j, nested_type_ptr, _buffer);
+                    if (WhichDataType(remove_nullable(nested_type_ptr)).is_string()) {
+                        buf_ret = _buffer.push_string("'", 1);
+                        buf_ret = _add_one_cell(data, j, nested_type_ptr, _buffer);
```

Comment:
What is the result if the nested data is null?

Reply:
```
mysql> set enable_vectorized_engine = true;
mysql> select k3 from test_array_string order by k2 limit 1;
+----------------------------------------------+
| k3                                           |
+----------------------------------------------+
| ['cdsdasd', 'xxxx', '3', '4', '5', NULL] |
+----------------------------------------------+
```
here is an example of nested data is null

Reply:
ok